To: vim_dev@googlegroups.com
Subject: Patch 8.1.1368
Fcc: outbox
From: Bram Moolenaar <Bram@moolenaar.net>
Mime-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit
------------

Patch 8.1.1368 (after 8.1.1366)
Problem:    Modeline test fails with python but without pythonhome.
Solution:   Correct test argument.
Files:	    src/testdir/test_modeline.vim


*** ../vim-8.1.1367/src/testdir/test_modeline.vim	2019-05-23 17:08:40.824565813 +0200
--- src/testdir/test_modeline.vim	2019-05-23 17:34:32.487469143 +0200
***************
*** 127,133 ****
    call s:modeline_fails('patchexpr', 'patchexpr=Something()', 'E520:')
    call s:modeline_fails('printexpr', 'printexpr=Something()', 'E520:')
    call s:modeline_fails('pythondll', 'pythondll=Something()', 'E520:')
!   call s:modeline_fails('pythonhome', 'pythondll=Something()', 'E520:')
    call s:modeline_fails('pythonthreedll', 'pythonthreedll=Something()', 'E520:')
    call s:modeline_fails('pythonthreehome', 'pythonthreehome=Something()', 'E520:')
    call s:modeline_fails('pyxversion', 'pyxversion=Something()', 'E520:')
--- 127,133 ----
    call s:modeline_fails('patchexpr', 'patchexpr=Something()', 'E520:')
    call s:modeline_fails('printexpr', 'printexpr=Something()', 'E520:')
    call s:modeline_fails('pythondll', 'pythondll=Something()', 'E520:')
!   call s:modeline_fails('pythonhome', 'pythonhome=Something()', 'E520:')
    call s:modeline_fails('pythonthreedll', 'pythonthreedll=Something()', 'E520:')
    call s:modeline_fails('pythonthreehome', 'pythonthreehome=Something()', 'E520:')
    call s:modeline_fails('pyxversion', 'pyxversion=Something()', 'E520:')
*** ../vim-8.1.1367/src/version.c	2019-05-23 17:08:40.824565813 +0200
--- src/version.c	2019-05-23 17:35:29.255134709 +0200
***************
*** 769,770 ****
--- 769,772 ----
  {   /* Add new patch number below this line */
+ /**/
+     1368,
  /**/

-- 
Shift happens.
                -- Doppler

 /// Bram Moolenaar -- Bram@Moolenaar.net -- http://www.Moolenaar.net   \\\
///        sponsor Vim, vote for features -- http://www.Vim.org/sponsor/ \\\
\\\  an exciting new programming language -- http://www.Zimbu.org        ///
 \\\            help me help AIDS victims -- http://ICCF-Holland.org    ///
